apps/portal/pages/opportunities/index.tsx

```diff
@@ -65,19 +67,24 @@ const ApplicationsPage: NextPage = () => {
         )}
       </header>
       <div className="w-full flex flex-wrap gap-[30px]">
-        {data.typeformApplications.map(({ id, typeformName, description, typeformId }) => (
-          <ApplicationCard
-            key={id}
-            title={typeformName}
-            description={description}
-            button={
-              <PopupButton id={typeformId} className="my-button">
-                <Button>apply</Button>
-              </PopupButton>
-            }
-            division="development."
-          />
-        ))}
+        {data.typeformApplications.map(
+          ({ id, typeformName, description, typeformId, externalResourceUrl }) => (
+            <ApplicationCard
+              key={id}
+              title={typeformName}
+              description={description}
+              buttons={[
+                <PopupButton id={typeformId} className="my-button">
+                  <Button>apply</Button>
+                </PopupButton>,
+                <Link href={externalResourceUrl} target="_blank">
```

Comment:
Can you make the "Learn More" button to show only when there is an external URL included?

Reply:
Will do!